Changes.md

```diff
@@ -5,7 +5,7 @@
 _TBC_
 
 ### Migration notes
-
+- [#3416](https://github.com/medic/medic-webapp/issues/3416) adds the mult_form_alerts transition to sentinel. See the documentation in [medic-docs](https://github.com/medic/medic-docs/blob/master/configuration/transitions.md) or in [kanso.json](https://github.com/medic/medic-webapp/blob/master/kanso.json).
```

Comment:
This isn't a migration note, it's a new feature. Add a section for features and put it there. Also remove the "or in kanso" bit - medic-docs has the best information in the easiest to follow format.